src/test/java/org/jenkinsci/plugins/gitclient/CredentialsTest.java

```diff
@@ -34,6 +34,7 @@
 import org.junit.After;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
```

Comment:
Same as the preceding comment.  Don't need to add this import, since the hamcrest assertions already provide the assertThat definition.
```suggestion
```